mobile/src/main/java/org/openhab/habdroid/model/OpenHABItem.java

```diff
@@ -151,7 +152,7 @@ public Float getStateAsFloat() {
 			try {
 				result = Float.parseFloat(state);
 			} catch (NumberFormatException e) {
-				e.printStackTrace();
+				Log.e(TAG, e.getMessage(), e);
```

Comment:
Not sure if that's really an error we should log as error.

Reply:
Probably not. In general I wouldn't log exceptions that occur only due to broken user/server input.

Reply:
I change their log level to d as it might be good to see, when those exceptions occur.